examples/generate.sh

```diff
@@ -80,14 +80,25 @@ fi
 
 mkdir -p "${OUTPUT_DIR}"
 
-# Generate Azure Credentials.
-Azure_B64ENCODED_CREDENTIALS="$(base64 -i "${GOOGLE_APPLICATION_CREDENTIALS}" | tr -d '\n')"
-export Azure_B64ENCODED_CREDENTIALS
+# Verify the required Environment Variables are present.
+: "${AZURE_SUBSCRIPTION_ID:?Environment variable empty or not defined.}"
+: "${AZURE_TENANT_ID:?Environment variable empty or not defined.}"
+: "${AZURE_CLIENT_ID:?Environment variable empty or not defined.}"
+: "${AZURE_CLIENT_SECRET:?Environment variable empty or not defined.}"
 
-Azure_REGION=${Azure_REGION:?}
-export Azure_REGION
-Azure_PROJECT=${Azure_PROJECT:?}
-export Azure_PROJECT
+
+AZURE_RESOURCE_GROUP=${AZURE_RESOURCE_GROUP:?}
+export AZURE_RESOURCE_GROUP
+
+AZURE_LOCATION=${AZURE_LOCATION:?}
+export AZURE_LOCATION
+
+# Azure Credentials.
+SSH_KEY_FILE=${OUTPUT_DIR}/sshkey
+export AZURE_SUBSCRIPTION_ID_B64="$(echo -n "$AZURE_SUBSCRIPTION_ID" | base64 | tr -d '\n')"
```

Comment:
@CecileRobertMichon -- not sure that we need `| tr -d '\n'`, as `echo -n` strips the newline.

Reply:
@vincepri suggested to add it https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/269#discussion_r328361741

Reply:
Sounds good!